packages/workbox-runtime-caching/test/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  globals: {
```

Comment:
This was a bit of a crutch as there are several files in this one directory that share a few globals, without them being used anywhere else.

There are similar issues throughout the codebase that are addressed via inline `/* eslint ... /*` config comments now, and I'm punting on addressing standardizing on that sort of thing for another PR.